README.md

```
@@ -0,0 +1,47 @@
+npm install
+npm run db:generate   # after schema changes
+npm run db:migrate
+npm run db:seed
+npm run dev           # http://localhost:8080
+```
+
+## GitHub Flow
+
+See [`CONTRIBUTING.md`](CONTRIBUTING.md) and the PR-sized backlog in [`docs/tasks/README.md`](docs/tasks/README.md).
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)